src/components/vm/consoles/consoles.jsx

```diff
@@ -88,7 +88,25 @@ class Consoles extends React.Component {
     onDesktopConsoleDownload (type) {
         const { vm } = this.props;
         // fire download of the .vv file
-        domainDesktopConsole({ name: vm.name, id: vm.id, connectionName: vm.connectionName, consoleDetail: vm.displays.find(display => display.type == type) });
+        const consoleDetail = vm.displays.find(display => display.type == type);
+
+        let address;
+        if (cockpit.transport.host == "localhost") {
+            const app = cockpit.transport.application();
+            if (app.startsWith("cockpit+=")) {
+                address = "wrong:" + app.substr(9);
```

Comment:
These 2 added lines are not executed by any test.

Reply:
The test failure shows the wrong value that is computed here. Hmm. Maybe the coverage collection goes wrong across logouts?